test.disabled/ejabberd_tests/tests/muc_SUITE.erl

```diff
@@ -83,7 +83,9 @@ all() -> [
         {group, room_management},
         {group, http_auth_no_server},
         {group, http_auth},
-        {group, hibernation}
+        {group, hibernation},
+        {group, register},
+        {group, register_over_s2s}
```

Comment:
there was a bug in previous version when we tried to determine ServerHost value based on UserJID.
Works when both muc and user are on the same host, but not over s2s.
Plus I needed a custom config for mysql with extra lines:

```erlang
{pool, odbc, coolpool, [{odbc_server, {mysql, "localhost", 33060, "ejabberd", "ejabberd", "ejabberd"}}, {odbc_server_type, odbc}]}.
{host_config, "localhost", [{odbc_pool, coolpool}]}.
```

TLDR:
```erlang
%% Works                                                                             
mongoose_rdbms:sql_transaction(<<"localhost">>, fun erlang:yield/0).                 
%% Fail                                                                              
mongoose_rdbms:sql_transaction(<<"fed.localhost">>, fun erlang:yield/0). 
 ```

But both works if we have a default mysql pool.